packages/components/view-switcher/src/view-switcher.styles.ts

```diff
@@ -0,0 +1,63 @@
+import { css } from '@emotion/react';
+import { customProperties as vars } from '@commercetools-uikit/design-system';
```

Comment:
Personally I think `customProperties` is a more meaningful name than just `vars`.
Perhaps we don't need to create an alias here.